python/lsst/ts/mtaos/mtaos.py

```diff
@@ -295,6 +295,17 @@ async def configure(self, config: typing.Any) -> None:
 
         self.log.debug("MTAOS configuration completed.")
 
+    async def end_enable(self, data):
+        """Runs after CSC goes into enable.
+
+        Parameters
+        ----------
+        data : `DataType`
```

Comment:
I think this might be `salobj.BaseMsgType`?
https://github.com/lsst-ts/ts_salobj/blob/develop/python/lsst/ts/salobj/type_hints.py#L36-L42